### DISCUSSION TITLE
champr_v2.0.2-b3 can't set game folder

### DISCUSSION THREAD

**Discussion by shaker3848** [cangzhang/ChampR#274] [General]
There is no button to set LOL game folder
![图片](https://github.com/cangzhang/champ-r/assets/30319371/0ea54da9-3caf-42c3-a146-8e7fcc815f37)

**Comment by cangzhang** [comment]
It's not necessary to specify LoL folder anymore, just start LoL and hit the rocket button, ChampR will automatically import builds to correct location
